src/Uno.UI/UI/Xaml/UIElement.Android.cs

```diff
@@ -261,6 +261,53 @@ View GetMatchInChildren(ViewGroup parent)
 			}
 		}
 
+		/// <summary>
+		/// Convenience method to find all views with the given name.
+		/// </summary>
+		public FrameworkElement[] FindViewsByName(string name) => FindViewsByName(name, searchDescendantsOnly: false);
```

Comment:
Internal APIs ?

Reply:
These are wrapped in a `#if DEBUG` condition